src/app/game-launcher.js

```diff
@@ -189,12 +201,14 @@ function showResult (playerState, chart) {
         'maxCombo':   stats.maxCombo,
         'accuracy':   stats.accuracy,
         'totalCombo': stats.totalCombo,
+        'totalNotes': stats.totalNotes,
         'log':        stats.log,
         'deltas':     stats.deltas,
         'grade':      getGrade(stats),
       },
-      chart:    chart,
+      chart: chart,
       playMode: playMode,
+      expertJudgmentWindow: player.notechart.expertJudgmentWindow,
```

Comment:
Maybe name this more explicitly as `lr2JudgmentWindow` so that it’s clear that it’s based off LR2.